apps/docs/components/package.json

```diff
@@ -16,8 +16,8 @@
     "@microsoft/api-documenter": "^7.25.9",
     "@microsoft/api-extractor": "^7.47.4",
     "@types/node": "^20.12.7",
-    "nuxt": "^3.12.4",
-    "nuxt-gtag": "^2.1.0"
+    "nuxt": "^3.13.2",
+    "nuxt-gtag": "^2.0.5"
```

Comment:
Is the downgrade on purpose?

Reply:
I havent changed this version of nuxt-gtag, weird, yarn had to change it, good catch will check it

Reply:
https://github.com/vuestorefront/storefront-ui/pull/3232/commits/79406be304830a435cdf725fd63edc7e29ab6290

Reply:
https://github.com/vuestorefront/storefront-ui/pull/3232/commits/79406be304830a435cdf725fd63edc7e29ab6290